pkg/cmd/server/start/start_master.go

```diff
@@ -245,6 +251,8 @@ func (o MasterOptions) RunMaster() error {
 		return nil
 	}
 
+	// Inject feature flags.
```

Comment:
Explain why its done here.  We know its injecting them, the weird part is that it happening after the config was written/read.  Explain that this is done because we want to disable certain features regardless of the values in the serialized config and we do not want to write the disabled features into the config because that hurts the upgrade path.